configure.ac

```diff
@@ -105,6 +106,15 @@ AC_CHECK_LIB(dl, dlopen, [],
 ])
 fi
 
+dnl handle picky option
+
+AC_ARG_ENABLE([picky],
+    [AC_HELP_STRING([--enable-picky],
+                    [Enable developer-level compiler pickyness when building (default: disabled)])])
+if test "$enable_picky" = "yes" -a "$GCC" = "yes" ; then
+  CFLAGS="$CFLAGS ${base_c_warn_flags} ${debug_c_warn_flags} -Wno-long-long -Wmissing-prototypes -Wstrict-prototypes -Wcomment -pedantic"
```

Comment:
Do you want to uniq-ify the CFLAGS?  E.g., if someone specifies `--enable-debug --enable-picky`, you'll get all the warning flags twice.  This is not *harmful*, per se, but it is untidy.

Perhaps only add `base_c_warn_flags` and `debug_c_warn_flags` if `$enable_debug` wasn't already yes, or perhaps unify the handling of `$enable_debug` and `$enable_picky`...?